react-front-end/tsrc/search/SearchPageModeReducer.ts

```diff
@@ -29,6 +30,12 @@ export type State =
       definition: OEQ.AdvancedSearch.AdvancedSearchDefinition;
       isAdvSearchPanelOpen: boolean;
       queryValues: FieldValueMap;
+      /**
+       * Indicates that the next Hide action should be ignored, so as to support
+       * initial searches and clearing of advanced searches. Will be reset to
+       * false on the next `action === hide`.
+       */
```

Comment:
No action required here (let's keep this PR going now), but in the next PR can you please expand this comment to say:

```suggestion
      /**
       * Indicates that the next Hide action should be ignored, so as to support
       * initial searches and clearing of advanced searches. Will be reset to
       * false on the next `action === hide`.
       *
       * This is due to interactions with the SearchPageReducer which this
       * reducer now probably needs to be merged with. Before the clear
       * button and the hiding of this pane based on the `search` action
       * in the SearchPageReducer it was independent, but not any-more.
       */
```

Reply:
Got it.